cli/src/main/java/io/zell/zdb/state/KeyFormatters.java

```diff
@@ -1,117 +0,0 @@
-/*
- * Copyright © 2021 Christopher Kujawa (zelldon91@gmail.com)
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.zell.zdb.state;
-
-import io.camunda.zeebe.db.DbValue;
-import io.camunda.zeebe.db.impl.*;
-import io.camunda.zeebe.protocol.ZbColumnFamilies;
-import java.util.HexFormat;
-import java.util.Map;
-import org.agrona.concurrent.UnsafeBuffer;
-
-public interface KeyFormatters {
-  KeyFormatter HEX_FORMATTER = new KeyFormatter.HexFormatter();
-  Map<ZbColumnFamilies, KeyFormatter> FORMATTERS =
-      Map.of(
```

Comment:
ℹ️ This was limited to 10 entries, afterwards we can only use a static block to initialze this, but this is not supported in interfaces. To make it work we had to refactor a bit the interface/classes.